apps/speed.c

```diff
@@ -547,21 +519,32 @@ static OPT_PAIR ecdsa_choices[] = {
     {"ecdsak283", R_EC_K283},
     {"ecdsak409", R_EC_K409},
     {"ecdsak571", R_EC_K571},
+
     {"ecdsab163", R_EC_B163},
     {"ecdsab233", R_EC_B233},
     {"ecdsab283", R_EC_B283},
     {"ecdsab409", R_EC_B409},
     {"ecdsab571", R_EC_B571},
-    {NULL}
+
+    {"ecdsabrp256r1", R_EC_BRP256R1},
+    {"ecdsabrp256t1", R_EC_BRP256T1},
+    {"ecdsabrp384r1", R_EC_BRP384R1},
+    {"ecdsabrp384t1", R_EC_BRP384T1},
+    {"ecdsabrp512r1", R_EC_BRP512R1},
+    {"ecdsabrp512t1", R_EC_BRP512T1}
 };
+# define ECDSA_NUM       OSSL_NELEM(ecdsa_choices)
 
-static OPT_PAIR ecdh_choices[] = {
+static double ecdsa_results[ECDSA_NUM][2];    /* 2 ops: sign then verify */
+
+static const OPT_PAIR ecdh_choices[] = {
     {"ecdhp160", R_EC_P160},
     {"ecdhp192", R_EC_P192},
     {"ecdhp224", R_EC_P224},
     {"ecdhp256", R_EC_P256},
     {"ecdhp384", R_EC_P384},
     {"ecdhp521", R_EC_P521},
+
```

Comment:
If additional empty line is meant to delimit prime and 2m fields, then how come there is no one prior brainpool chunk?